src/content/community/releases/_index.en.md

```diff
@@ -12,6 +12,7 @@ weight = 40
   restart of the CoreDNS server pod.
 * Fixed an issue with the OVN-Kubernetes CNI where, after a cluster recovery, the data path was broken requiring manual deletion of
   stale GatewayRoute and NonGatewayRoute resources and a restart of the Route Agent pod.
+* Fixed a crash in the Submariner Operator pod due to a concurrent map write.
```

Comment:
Shall we mention that this will fix the potential crash which can be seen during re-installation?

Reply:
Theoretically it's not specific to re-installation so I don't think we need to mention that. We do have multiple controllers running in the operator that access this code so it could happen anytime but it requires precise timing.